File: app/src/main/java/com/example/rightnow/MainActivity.java
package com.example.rightnow;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    private Button button;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        button = findViewById(R.id.log);
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                openActivityLoginPage();
            }
        });
    }

    public void openActivityLoginPage() {
        Intent intent = new Intent(this, loginPage.class);
        startActivity(intent);
    }
}

File: app/src/main/java/com/example/rightnow/MainActivity.kt
package com.example.rightnow

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    private var button: Button? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        button = findViewById<Button>(R.id.log)
        button.setOnClickListener(View.OnClickListener { openActivityLoginPage() })
    }

    fun openActivityLoginPage() {
        val intent = Intent(this, loginPage::class.java)
        startActivity(intent)
    }
}